### PR TITLE
Update build, Take Two

### PR DESCRIPTION
**Pull request by bonnland**

### CODE REVIEW

File: .github/workflows/binderbot.yaml [hunk 1]
```diff
@@ -52,25 +55,29 @@ jobs:
...
         run: |
           git config --local user.email "action@github.com"
           git config --local user.name "GitHub Action"
           git add binder-gallery.yaml thumbnail.png *.ipynb
           git commit -m "Binderbot output"
       - name: Push commit
-        if: github.ref == 'refs/heads/master'
-        uses: ad-m/github-push-action@master
+        if: github.ref == 'refs/heads/main'
+        uses: ad-m/github-push-action@main
```
Comment (kmpaul) [inline .github/workflows/binderbot.yaml]: This should fix the action.

```suggestion
        uses: ad-m/github-push-action@master
```

**Review by kmpaul — CHANGES_REQUESTED** [review]
Commit this and it should run the binderbot action, now.

**Comment by bonnland** [comment]
OK, binderbot build is running after that commit.   If it fails, I'm done for today.  I've been confused for many hours and need a break.

**Comment by bonnland** [comment]
Mentioning https://github.com/pangeo-gallery/pangeo-gallery/pull/48 here in case merging that PR will fix the 500 ISE.

**Comment by kmpaul** [comment]
Yeah. That seems to be the blocker. Thanks for getting this to this state, @bonnland!

**Comment by rabernat** [comment]
Once you merge this, things should work.

**Comment by rabernat** [comment]
> Mentioning [pangeo-gallery/pangeo-gallery#48](https://github.com/pangeo-gallery/pangeo-gallery/pull/48) here in case merging that PR will fix the 500 ISE.

No that is not the right interpretation. This PR needs to be merged FIRST, even with the current binderbot error (which, as I said several times, is ignorable, not something you need to worry about). 

That error comes up because this gallery is not yet part of pangeo-gallery (because pangeo-gallery/pangeo-gallery#48  has not yet been merged). This workflow is trying to trigger a rebuilt action on the pangeo-gallery website, but it's failing because pangeo-gallery doesn't have this repo as a submodule (yet).

However, we can't add the submodule to pangeo-gallery until the binderbot workflow has been _run on the `main` branch` and created a `binderbot-build` branch`. This is the chicken-and-egg situation I referred to.

The solution is to 
1. merge this PR and have binderbot create the `binderbot-built` branch
2. merge pangeo-gallery/pangeo-gallery#48  

in that order.

I'm sorry that the process I have created is so confusing and that it is causing stress. I offer to meet with you to personally talk you through how binderbot and pangeo-gallery work:
https://calendly.com/ryan-abernathey/15min 

More generally, I welcome your inputs on binderbot, pangeo-gallery, and their documentation. There are a great many ways it could be improved.

**Comment by bonnland** [comment]
Thanks for the offer; I don't think we need to meet about it after your latest explanation.   I realize these things have some inherent complexity that will seem natural and obvious eventually.   In this case, it was the (now obvious) issue about which branch I was running the workflow on.   I didn't even consider that as making a difference at first.

**Review by kmpaul — APPROVED** [review]
Given @rabernat's comment, I approve this merge.  It looks like it's ready.

**Pull request merged**